File: python/notes/conditionals.py
#CW Conditionals

username = input("What is your username: ").strip().lower()

number = int(input("Provide a number: ").strip())

if number < 10 and number >= -9:
    print(f"{number} is a single digit number")
elif number < 100 and number >= -99:
    print(f"{number} is a double digit number")
else: 
        print(f"{number} is a triple digit number")
        new_number = number*number
        if new_number > 1000:
             print(f"Suprise number! {new_number}")

if username == "lupus":
    print("Welcome Admin: lupus")
else: 
    print(f"Welcome User: {username}")

print("END OF LINE")

File: C/notes/conditionals.c
//CW Conditionals Notes
#include <stdio.h>
#include <string.h>
#include <stdbool.h>

int grade = 90;
int num = -3;
char name[] = "Lupus";

int main(){

    if(grade > 90){
        if(grade > 93){
            printf("Your grade is an A\n");
        }
        printf("Your grade is an A-\n");
    } else if (grade > 80) {
        printf("Your grade is a B\n");
    } else if (grade > 70){
        printf("Your grade is an A\n");
    } else if(grade > 60){
        printf("Your grade is an A\n");
    } else if(grade > 50){
        printf("Your grade is an A\n");
    } else{
        printf("You are failing\n");
    }

    if(num < 10 && num > -10){
        printf("Your number is %d, that is a single digit number\n", num);
    } else if( num < 100 && num > 9){
        printf("Your number is %d, that is a double digit number.\n", num);
    } else {
        printf("Your number is %d, that is a very big number\n", num);
    }

    if (strcmp(name, "Lupus") == 0){
        printf("Hello WOLF_LORD\n");
    } else{
        printf("Hello new user: %s\n", name);
    }

    return 0;
}
